frontend/src/features/create-simple-collection/types.ts

```diff
@@ -3,11 +3,11 @@ type SummaryValues = {
   logo: File | undefined;
   name: string;
   description: string;
-  url: string;
-  telegram: string;
-  x: string;
-  medium: string;
-  discord: string;
+  url?: string | undefined;
+  telegram?: string | undefined;
+  x?: string | undefined;
+  medium?: string | undefined;
+  discord?: string | undefined;
```

Comment:
Even if those fields are optional, I think original types make more sense, because if field is empty - it returns empty string. RHF is sometimes unpredictable with nullable values.